files/chef-marketplace-cookbooks/chef-marketplace/libraries/helpers.rb

```diff
@@ -34,14 +34,27 @@ def current_sudoers
       Dir['/etc/sudoers.d/*']
     end
 
-    def gecos
+    def cloud_cfg_gecos
       case node['chef-marketplace']['platform']
       when 'aws' then 'Ec2 User'
+      when 'azure' then 'Azure User'
       when 'openstack' then 'OpenStack User'
       when 'oracle' then 'Oracle User'
       end
     end
 
+    def cloud_cfg_ssh_pwauth
+      node['chef-marketplace']['platform'] == 'azure' ? true : false
+    end
+
+    def cloud_cfg_locale_configfile
+      node['platform_family'] == 'rhel' ? '/etc/sysconfig/i18n' : '/etc/default/locale'
+    end
+
+    def cloud_cfg_distro
+      node['platform_family'] == 'rhel' ? 'rhel' : 'ubuntu'
```

Comment:
This feels wrong to just assume anything not rhel is ubuntu even though we're controlling it. Feels like the kind of thing that could blow up in the future when we go to support another OS. 

Return it if it's rhel or Ubuntu, raise otherwise?